### DISCUSSION TITLE
Rust in NNStreamer/NNTrainer

### DISCUSSION THREAD

**Discussion by myungjoo** [nnstreamer/nnstreamer#3912] [General]
Let's discuss the possible usage and adoptation of Rust in NNStreamer/NNTrainer.

Resources
- https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs

**Comment by rismanadnan** [comment]
This is good idea for future roadmap. In AI community, we started to see rust adoption such as rust BERT (https://docs.rs/rust-bert/latest/rust_bert/). We can embed C++ codes directly using rust-cpp.
